src/main/java/spoon/refactoring/CtRenameLocalVariableRefactoring.java

```diff
@@ -84,9 +84,8 @@ public void accept(CtReference t) {
 
 		@Override
 		public ScanningMode enter(CtElement element) {
-			if (ignoredParent != null && element instanceof CtElement) {
-				CtElement ele = (CtElement) element;
-				if (ele.hasParent(ignoredParent)) {
+			if (ignoredParent != null && element != null) {
+				if (element.hasParent(ignoredParent)) {
```

Comment:
Thank you, it is really better now. Would not it be even easier, if there would be only one `if` and longer condition?